pkg/interop/util/util.go

```diff
@@ -36,3 +38,19 @@ func Equals(a, b interface{}) bool {
 func Remove(slice interface{}, i int) {
 	neogointernal.Opcode2NoReturn("REMOVE", slice, i)
 }
+
+// CallWithVersion is a utility function that xecutes the previously deployed
+// blockchain contract with the specified version (update counter) and hash
+// (20 bytes in BE form) using the provided arguments and call flags. It fails
+// if the contract has version mismatch. It returns whatever this contract
+// returns. This function uses `System.Contract.Call` syscall.
+func CallWithVersion(scriptHash interop.Hash160, version int, method string, f contract.CallFlag, args ...interface{}) interface{} {
```

Comment:
Move it to `pkg/interop/lib/contract`? `util` just wraps some opcodes.

Reply:
Yes, fixed.